actionlib/include/actionlib/managed_list.h

```diff
@@ -137,6 +137,15 @@ class ManagedList
       return *this;
     }
 
+    Handle(const Handle& rhs)
+    {
+      if (rhs.valid_) {
```

Comment:
alternative would be to reuse the reviewed operator= code.

Reply:
That's what it does? Or do you mean `*this = rhs`?

Reply:
yup. `*this = rhs`.

Reply:
I like that. Changed it to be `*this = rhs` :+1: